solidity/contracts/token/ERC223SmartToken.sol

```diff
@@ -1,25 +1,485 @@
 pragma solidity ^0.4.23;
 
-import "./SmartToken.sol";
+contract DSAuthority {
```

Comment:
Why not use https://github.com/evolutionlandorg/common-contracts/blob/master/contracts/StandardERC223.sol ?